### ISSUE TITLE
Add accessibility link back

### ISSUE THREAD

**Issue by josepjaume** [AjuntamentdeBarcelona/decidim.barcelona#635]
We're missing the accessibility link on the footer:

http://ajuntament.barcelona.cat/accessibilitat/ca
